.github/workflows/container-release.yml

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright 2025 The Sigstore Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+name: Release Services Build Container
+
+on:
+  push:
+    tags:
+    - 'v*'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    permissions:
+      id-token: write
+      contents: read
+      packages: write
+      attestations: write
+
+    env:
+      REGISTRY: ghcr.io
+      IMAGE_NO_TAG: ghcr.io/${{ github.repository }}-service-builder
+      IMAGE_REF: ghcr.io/${{ github.repository }}-service-builder:${{ github.ref_name }}
+
+    steps:
+    - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+    
+    - name: Log into ghcr.io
+      uses: docker/login-action@74a5d142397b4f367a81961eba4e8cd7edddf772 # v3.4.0
+      with:
+        registry: ${{ env.REGISTRY }}
+        username: ${{ github.repository_owner }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Build services container
+      run: make services-image
+
+    - name: Tag image
+      run: docker tag protoc-services ${{ env.IMAGE_REF }}
+
+    - name: Push image
+      run: docker push ${{ env.IMAGE_REF }}
+
+    - name: Get image digest
+      run: |
+        digest=$(docker inspect --format='{{index .RepoDigests 0}}' ${{ env.IMAGE_REF }})
+        image_digest=$(echo $digest | cut -d"@" -f2)
+        echo "IMAGE_DIGEST=$image_digest" >> "$GITHUB_ENV"
+
+    - name: Generate artifact attestation
```

Comment:
If you use build-push-action, it gives you the container digest - https://github.com/sigstore/rekor-tiles/blob/9d115da47b205fc2a8fc6d7e369f9f3d5a2556ee/.github/workflows/build_container.yml#L119-L125

Reply:
That action wants to do the build. There's no way to skip it

Reply:
Ah yea, this looks good then.